pages/index.html

```diff
@@ -4,9 +4,34 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel = "stylesheet" href = "styles/styles.css">
   <title>Document</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Shadows+Into+Light&display=swap" rel="stylesheet">
 </head>
-<body>
-  
+<body class="color">
+  <nav>
+    <ul>
+      <li>
+          <a class= "decoration" href = "portfolio.html">PORTFOLIO</a>
+      </li>
+      <li>
+        <a class= "decoration" href = "index.html">INDEX</a>
+    </li>
+      <li>
+          <a class= "decoration" href = "about.html">ABOUT</a>
+      </li>
+    </ul>
+  </nav>
+  <center><h1>LILIAN ASHLEY MORA</h1></center>
+  <div class="flex-container">
+
+    <div>FOODIE</div>
+    
+    <div>TRAVEL</div>
+    <div>MUSIC</div>
```

Comment:
`div` is used to divide elements, not display text. So let's put FOODIE and such inside a heading or paragraph tag _inside_ the `div`. Or even better, replace `div` with `section`